<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvnCleanupDialog</class>
 <widget class="QWidget" name="SvnCleanupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>182</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SVN Cleanup...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Clean up directory:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEditDirectory">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBoxUnversioned">
     <property name="text">
      <string>Delete unversioned files and directories</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxIgnored">
     <property name="text">
      <string>Delete ignored files and directories</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkBoxExternals">
     <property name="text">
      <string>Include externals</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="buttonOk">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-ok">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-cancel">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
